### ISSUE TITLE
Number of epochs

### ISSUE THREAD

**Issue by flaviuv** [shaoyuanlo/EDANet#5]
Hi, I am trying to reproduce the results in the paper so I am wondering if you can share the number of epochs you used to train train the model. Thanks for your time.

**Comment by shaoyuanlo** [comment]
The models for ablation study (Table 1) are trained by 150 epochs.
The model for the final evaluation (Table 2) is trained by two-stage. The first stage has 150 epochs, and the second stage has 300 epochs.

**Comment by CS-Jackson** [comment]
Hi, @shaoyuanlo . What do you mean two-stage? Did you train the model for 150 epochs and then train the model for other 300 epochs using a restart learning rate? Or you train the model for totally 450 epochs?

**Comment by shaoyuanlo** [comment]
Hi @CS-Jackson 
Yes, your statement is correct.
But please note that the first stage is trained by downsampled annotations (1/8 resolution, i.e., do not upsample the feature maps), and the second stage is trained by full resolution (feature maps are upsampled 8x to the input resolution by bilinear interpolation).

**Comment by CS-Jackson** [comment]
@shaoyuanlo . The 1/8 resolution is for the origin sample(1024, 2048), or for the training sample(512, 1024)?

**Comment by shaoyuanlo** [comment]
@CS-Jackson   for the training sample (512,1024)
